langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -99,6 +117,18 @@ public isolated function get(xml<ItemType> x, int i) returns ItemType = @java:Me
 } external;
 
 # Concatenates xml and string values.
+# 
+# ```
+# // Concatanate a set of xml variables
+# xml firstBook = xml `<book>Ballerina</book>`;
+# xml secondBook = xml `<book>Java</book>`;
+# xml thirdBook = xml `<book>Python</book>`;
+# xml bookSet = xml:concat(firstBook, secondBook, thirdBook);
+# 
+# // Concatanate an array of xml values
+# xml[] subjectList = [xml `<subject>English</subject>`, xml `<subject>Math</subject>`, xml `<subject>ICT</subject>`];
+# xml subjects = xml:concat(...subjectList);
```

Comment:
It is better to add a usage example like
`firstBook.concat(secondBook)`,
because it is a useful scenario to concat xml into exsiting xml value.

Reply:
Updated in: https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/240eeaad04c7ba4dc901592cdc6875a656057250